### PR TITLE
start on nagios 7 upgrade, change versions

### PR DESCRIPTION
**Pull request by rettigs**

### CODE REVIEW

**Comment by rettigs** [comment]
@jordane Just to make sure, deleting the `Berksfile.lock` and then doing `berks upload` shouldn't break anything right?  It just uploads new versions of cookbooks, but the environments that need specific versions should already have them pinned, yes?

**Comment by jordane** [comment]
@rettigs Yes, that should be fine. We don't usually even commit `Berksfile.lock` in cookbook repos..

File: Berksfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source 'https://supermarket.getchef.com'
 
 cookbook "munin"
-cookbook "nagios", git: "github.com:osuosl-cookbooks/nagios"
+cookbook "nagios", version: "~> 7.0"
```
Comment (jordane) [inline Berksfile]: Is there a reason you're using a pessismistic lock here? It will just cause problems later.

Comment (rettigs) [inline Berksfile]: I wanted to constrain it to use version 7.x.x.  What problems do you anticipate?

Comment (jordane) [inline Berksfile]: As soon as `8.x.y` is released this will stick to 7, which will just be annoying. Might as well use `>=` or no constraints at all.

**Comment by jordane** [comment]
@rettigs Merging this into master probably won't go well while both the Nagios 7.x.x and 4.x.x cookbooks live in Chef together. A better solution is to make a `1.x` branch and merge it in there, then eventually merge that into master when we're ready to kill off the `0.x` series.

**Comment by rettigs** [comment]
@jordane What exactly do you mean?  Why wouldn't it go well?  And isn't this already a 1.0 branch?

**Comment by jordane** [comment]
@rettigs It wouldn't go well because if we have to make changes to our fork before this is done, we essentially have to maintain 2 'master' branches. You might want to read up on [gitflow](http://nvie.com/posts/a-successful-git-branching-model/)

File: metadata.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 name             'monitoring'
 maintainer       'Oregon State University'
-maintainer_email 'rudy@grigar.net'
+maintainer_email 'systems@osuosl.org'
```
Comment (ramereth) [inline metadata.rb]: This should be chef@osuosl.org.

**Comment by armiller** [comment]
Should we close this since we are deprecating the monitoring cookbook?

**Pull request closed without merging**